From 09b4d1aea3e0d5bc12d255da680f2c7a9f6c4bd3 Mon Sep 17 00:00:00 2001
From: Chris Morgan <macromorgan@hotmail.com>
Date: Wed, 30 Oct 2024 20:46:46 -0500
Subject: [PATCH 05/10] arm64: dts: rockchip: Correct pmu_io_domain regulators
 for RGxx3

Correct the pmu_io_domain regulators for the Anbernic RGxx3 series.
These domains were indivually tested by identifying a test point on the
board that corresponded to a pin within the power domain as stated in
section 2.7 of the rk3566 datasheet. The pin was then set to a GPIO
output of 1, and the voltage of the regulator was slightly adjusted up
or down until the changes matched what was observed on the pin.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
index b5d7096ffbcc..51d42421dad0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
@@ -566,10 +566,11 @@ &pmu_io_domains {
 	pmuio1-supply = <&vcc3v3_pmu>;
 	pmuio2-supply = <&vcc3v3_pmu>;
 	vccio1-supply = <&vccio_acodec>;
+	vccio2-supply = <&vcc_1v8>;
 	vccio3-supply = <&vccio_sd>;
-	vccio4-supply = <&vcc_1v8>;
+	vccio4-supply = <&vcc1v8_dvp>;
 	vccio5-supply = <&vcc_3v3>;
-	vccio6-supply = <&vcc1v8_dvp>;
+	vccio6-supply = <&vcc_1v8>;
 	vccio7-supply = <&vcc_3v3>;
 };
 
-- 
2.43.0

